packages/web-comment/src/utils/appealSorting.js

```diff
@@ -0,0 +1,36 @@
+const currentDate = new Date();
+const getOpenAppeals = (appeals) => {
+	return appeals.filter((appeal) => new Date(appeal.interestedPartyRepsDueDate) > currentDate);
+};
+
+const getClosedAppeals = (appeals) => {
+	return appeals.filter((appeal) => new Date(appeal.interestedPartyRepsDueDate) < currentDate);
+};
+
+// should we use this type of function declaration?
+Array.prototype.sortByInterestedPartyRepsDueDate = function () {
```

Comment:
Generally bad practice to alter prototypes - a reason this isn't just a normal sort function? Something like:

```
function sortByInterestedPartyRepsDueDate(a, b) {
  return new Date(a.interestedPartyRepsDueDate) - new Date(b.interestedPartyRepsDueDate);
}

myArr.sort(sortByInterestedPartyRepsDueDate);`
```

Reply:
agreed. Have changed this.